javaOOP_inheritance/src/jv/oop/inheritance/exercise/animals/AnimalsFactoryInt.java

```diff
@@ -0,0 +1,5 @@
+package jv.oop.inheritance.exercise.animals;
+
+public interface AnimalsFactoryInt {
```

Comment:
I would name the interface as AnimalsFactory and the implementation class as AnimalsFactoryImpl. But it doesn't matter so much.

Reply:
Okay